### ISSUE TITLE
Migrator from node-cryptonote-pool

### ISSUE THREAD

**Issue by Snipa22** [Snipa22/nodejs-pool#24]
My intention is to move balances, is there anything else people would like moved into LMDB?

**Comment by bapparabi** [comment]
yes i have node-cryptonote-pool and i an switch  to you if you tranfer eve thing and also i need to know do it has the option by default miner to exchange and alo put worker if and rig name ??

**Comment by Thomasmcdaniel9** [comment]
How hard would it be to transfer a pool thats got miners and is in use to transfer to the nodejs pool and keep everyone from thinking the pool has a problem or would we need to start from scratch and have all of our miners move. because we are getting request miners want to be able to mine directly to exchanges and be able to monitor there miners because the  cryptonote pool does not support that

**Comment by sevendoorknobs** [comment]
Hey Snipa, any update on this process?
